tools/docker/builder/openwrt/Dockerfile

```diff
@@ -67,30 +68,13 @@ COPY --chown=openwrt:openwrt configs/prplmesh_config $HOME/prplmesh_config
 ARG PRPLMESH_VERSION=82f1e83
 ARG TARGET_PLATFORM_TYPE=turris-omnia
 ARG PRPL_FEED=https://git.prpl.dev/prplmesh/feed-prpl.git
```

Comment:
Could you add a documentation comment for these three as well?

Reply:
Done.